sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/airtable.yaml

```diff
@@ -0,0 +1,104 @@
+provider:
+  metadata:
+    name: Airtable API key
+    category: Cloud Collaboration
+    message: Make sure this Airtable API key gets revoked, changed, and removed from the code.
+  detection:
+    pre:
+      include:
+        content:
+          - airtable
+
+  rules:
+    - id: airtable-api-key
+      rspecKey: S6733
+      metadata:
+        name: Airtable API keys should not be disclosed
+      detection:
+        matching:
+          pattern: "(?i)\\bkey[\\w]{14}\\b"
+          context:
+            patternAround: "(?i)(airtable|base|table)"
+        post:
+          #patternNot:
+          #  - "(?i)(s|ex)ample"
+          #  # keyXXXXXXXXXXXXXX and key00000000000000
+          #  - "(\\w)\\1{4,}"
+          #  # keyAPI12345678910
+          #  - "12345678"
+          patternNot: "(?i)\
+            example|\
+            (\\w)\\1{4,}|\
+            12345678"
+      examples:
+        - text: |
+            // rspec
```

Comment:
I think this should be:
`props.set("airtable_key", "key6yLyCekATg67Ts") `
So that it really is the example from rspec.